src/pages/en/guides/styling.md

```diff
@@ -92,6 +92,43 @@ const backgroundColor = "rgb(24 121 78)";
 
 📚 See our [directives reference](/en/reference/directives-reference/#definevars) page to learn more about `define:vars`.
 
+### Passing a `class` to a child component
+
+In Astro, HTML attributes like `class` do not pass through to a child component unless they are used as props.
+
+```astro title="src/pages/index.astro"
+---
+import MyComponent from "../components/MyComponent.astro"
+---
+<MyComponent class="red">Color me!</MyComponent>
```

Comment:
I was thinking maybe we can add the style tag to make it more obvious.

```suggestion
<style>
  .red {
    color: red;
  }
</style>
<!-- this won't color the text to red -->
<MyComponent class="red">Color me!</MyComponent>
```

Reply:
This makes sense, thanks!